docs/admin/backup_recovery.md

```diff
@@ -20,6 +20,10 @@ description: Data Backup and recovery
 
 ## {{site.data.keyword.open_shift_cp}} backup and recovery
 
+For more information about cluster-wide data backup and recovery, see:
+
+* [{{site.data.keyword.open_shift_cp}} 4.6 backup etcd](https://docs.openshift.com/container-platform/4.6/backup_and_restore/backing-up-etcd.html).
```

Comment:
We may want to leave this out, or find a K8s-neutral source instead, or find comparable sources for all K8s clusters that OH supports.

Reply:
This commit syncs the OH / IEAM pages.  I didn't validate the url (which is broken, now fixed in another commit)

Do you want me to back out the backup/recovery etcd link?

Reply:
Yes, let's back this out for now, and open a separate issue to track.

Reply:
If you read the entire page, its really about OpenShift. The entire page should be removed.  That should be part of the conversation in #221.  The work in this #130 issue  just syncs the deltas.  I didn't try to pass judgement on worthiness.

Let's punt the conversation to #221